src/featuredetection.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module utils/featuredetection
+ */
+
+/**
+ * Holds feature detection used by the editor.
+ *
+ * @protected
+ * @namespace
+ */
+export default {
```

Comment:
I was wondering whether it should be palced in https://github.com/ckeditor/ckeditor5-utils/blob/cf06347de71b235b564a13016a260af7c191a34d/src/unicode.js - but at the end the linked file is related to unicode handling, while this check is focused on RegExp in context of a given unicode goup handling.

However I'd give it better namespacing, I'm thinking more like: `utils.featuredetection.regexp.isXyzSupported()`.

Reply:
@mlewand  to clarify:
```js
import featureDetection from '@ckeditor5-utils/src/utils/featuredetection';

console.log(
    'Regex group is supported:',
    featureDetection.regex.isUnicodePropertySupported()
);
```

ps.: Shouldn't it be `isUnicodePropertyEscapesSupported()`? :laughing:

Reply:
@msamsel please create also a PR in mention repo to use this util instaed own feature detection.

Reply:
@jodator I was thinking more about:

```js
import regExpFeatureDetection from '@ckeditor5-utils/src/utils/featuredetection/regexp';

console.log(
    'RegExp group is supported:',
    regExpFeatureDetection.isUnicodePropertySupported()
);
```

This would allow us to nicely organize stuff in future.

And it's important for us to use RegExp with **p** consistently, as this is how the type is named in JS.

Reply:
>  RegExp with **p** consistently, a

typo, sorry :P